.gitignore

```diff
@@ -11,3 +11,5 @@ buildNumber.properties
 .project
 .settings/
 .checkstyle
+.idea/
+*.iml
```

Comment:
Please back out this change since it's only needed for developers  that use a particular IDE. These entries should go in your personal <git_user_home>/git_ignore file (for me that file is C:\Users\Chris\Git\git_ignore). I have some eclipse specific entries in there since I use eclipse.